src/Agent.Service/Windows/AgentService.csproj

```diff
@@ -12,9 +10,13 @@
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>FinalPublicKey.snk</AssemblyOriginatorKeyFile>
     <DelaySign>true</DelaySign>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
I appreciate the effort to modernize the service.  However, I this is potentially the type of change we should consider flighting / rolling out safely.  Leaving a comment so we can discuss